### DISCUSSION TITLE
multiple select column type => tags

### DISCUSSION THREAD

**Discussion by Rhydderch** [RafaelGB/obsidian-db-folder#542] [Q&A] [answered]
### Present your request

I would like to be able to select multiple values when the column type is "select".
Alternatively, it could be a different column type called "Select multiple".

The select column is super useful but making it "multiple select" would make it even more versatile!

### For which platform do you request this request??

Cross

**Comment by DannyHatcher-Git** [comment, marked as answer]
The multi-select option is the tags property. 
Options in there only become tags if the property id says tags, otherwise, it acts as a multi-select.

**Reply by Rhydderch** [reply]
Thanks a lot! I wrongly thought it would only work with tags (due to its name).

**Reply by DannyHatcher-Git** [reply]
I explain the properties in more detail in the video I posted on YouTube 😁

**Reply by Rhydderch** [reply]
I'll go check your videos 😊 !
